networklibrary/src/main/java/com/eis/smsnetwork/SMSNetworkManager.java

```diff
@@ -0,0 +1,96 @@
+package com.eis.smsnetwork;
+
+import com.eis.communication.network.FailReason;
+import com.eis.communication.network.Invitation;
+import com.eis.communication.network.NetworkManager;
+import com.eis.communication.network.listeners.GetResourceListener;
+import com.eis.communication.network.listeners.InviteListener;
+import com.eis.communication.network.listeners.RemoveResourceListener;
+import com.eis.communication.network.listeners.SetResourceListener;
+import com.eis.smslibrary.SMSPeer;
+
+/**
+ * The manager class of the network.
+ *
+ * @author Edoardo Raimondi
+ * @author Marco Cognolato
+ */
+public class SMSNetworkManager implements NetworkManager<String, String, SMSPeer, FailReason> {
+
+    private SMSNetSubscriberList netSubscribers = new SMSNetSubscriberList();
+    private SMSNetDictionary netDictionary = new SMSNetDictionary();
```

Comment:
Could be `NetDictionary<String, String>`.

Reply:
Same as the NetSubList, SMSNetDictionary implements NetDictionary<String,String>